omod/src/main/webapp/pages/login.gsp

```diff
@@ -175,7 +175,7 @@
                                     onclick="javascript:window.location = '/${ contextPath }/logout'"
                                     value="${ ui.message("general.cancel") }" />&nbsp;&nbsp;
                             <% } %>
-                                <input id="loginButton" class="btn ${ ui.message(selectLocation ? "confirm" : "btn-success") }" type="submit"
+                                <input id="loginButton" class="btn ${ ui.message(selectLocation ? "btn-success" : "confirm") }" type="submit"
```

Comment:
Oh. So you just reversed them? Did you test and confirm?

Reply:
yes, i watched the module with a given running server and it worked ..
Also tested it out on the `qa server` while making some changes on inspecting code in the `Log In` button and it works